File: proxy/v1/interaction/read-default/read-default.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Proxy.V1.Service.Session;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var interactions = InteractionResource.Read(
            pathServiceSid: "KSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathSessionSid: "KCXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            limit: 20
        );

        foreach(var record in interactions)
        {
           Console.WriteLine(record.Sid);
        }
    }
}

File: taskrouter/v1/worker_channel/read-default/read-default.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

worker_channels = @client.taskrouter
                         .workspaces('WSXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                         .workers('WKXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                         .worker_channels
                         .list(limit: 20)

worker_channels.each do |record|
  puts record.sid
end
